hack/hive/hive-config/generate.go

```diff
@@ -3,4 +3,4 @@ package hiveconfig
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
-//go:generate go run ../genhiveconfig ./hive-additional-install-log-regexes.yaml
+//go:generate go run ../../genhiveconfig ./hive-additional-install-log-regexes.yaml
```

Comment:
Probably moving the `genhiveconfig` directory under `hack/hive/` would be a good idea for grouping Azure resources.